proseco/acq.py

```diff
@@ -1346,8 +1393,14 @@ def get_imposter_stars(
     # contiguous regions above ``thresh`` labeled with a unique index.
     # This is a one-line way of doing the PEA merging process, roughly.
     dc_labeled, n_hits = ndimage.label(dc2x2 > thresh)
+    if ACQ.search_hit_max_size is not None:
+        dc_labeled, n_hits = split_labeled_regions(
+            dc_labeled, max_size=ACQ.search_hit_max_size
```

Comment:
This is throwing "ValueError: zero-size array to reduction operation maximum which has no identity" on my first toy test at `slices = ndimage.find_objects(dc_labeled)`, so unless you want to handle the empty case in split_labeled_regions, I think the "if n_hits == 0: return" code should probably come before split_labeled_regions.

Reply:
Can you provide a reproducible example? I tried the following and it worked as expected.
```
dc_labeled = np.array([[0, 0], [0, 0]])
pracq.split_labeled_regions(dc_labeled)
```
I agree that the `if n_hits == 0:` check could be moved up for efficiency.

Reply:
It looks like my original issue was that I both wasn't sure about the domain of "star_row" and "star_col" and I'm a bit flummoxed about why "test" changes the offset by 512 pixels.  It looks like now I'm only hitting ValueError if I'm out of bounds.

```
dark = mica.archive.aca_dark.get_dark_cal_image("2025:001", t_ccd_ref=-10)
dat = Table(get_imposter_stars(dark=dark, star_row=200, star_col=2000, box_size=(1000,1000)))
```